pandas/tests/frame/test_subclass.py

```diff
@@ -570,3 +570,56 @@ def strech(row):
         result = df.apply(lambda x: [1, 2, 3], axis=1)
         assert not isinstance(result, tm.SubclassedDataFrame)
         tm.assert_series_equal(result, expected)
+
+    def test_frame_subclassing_and_inherit(self):
```

Comment:
what is the purpose of these tests? this is another issue, yes?

Reply:
this should be in a new PR

Reply:
I wrote it to check that that approach would work now. I'll take it out.